packages/frontend/src/pages/SavedSearches/SavedSearchesPage.tsx

```diff
@@ -1,32 +1,30 @@
-import type { SavedSearchesFieldsFragment } from 'bff-types-generated';
-import { useRef, useState } from 'react';
+import { type BookmarksListItemProps, BookmarksSection } from '@altinn/altinn-components';
+import { useQueryClient } from '@tanstack/react-query';
+import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { deleteSavedSearch, updateSavedSearch } from '../../api/queries.ts';
 import { useParties } from '../../api/useParties.ts';
 import { PartyDropdown } from '../../components';
+import type { Filter } from '../../components';
+import { QUERY_KEYS } from '../../constants/queryKeys.ts';
 import { useFormatDistance } from '../../i18n/useDateFnsLocale.tsx';
-import { ConfirmDeleteDialog, type DeleteSearchDialogRef } from './ConfirmDeleteDialog/ConfirmDeleteDialog.tsx';
-import {
-  EditSavedSearchDialog,
-  type EditSavedSearchDialogRef,
-} from './EditSavedSearchDialog/EditSavedSearchDialog.tsx';
-import { SaveSearchesActions } from './SavedSearchesActions/SavedSearchesActions.tsx';
-import { SavedSearchesItem } from './SavedSearchesItem/SavedSearchesItem.tsx';
 import { SavedSearchesSkeleton } from './SavedSearchesSkeleton';
 import styles from './savedSearchesPage.module.css';
 import { autoFormatRelativeTime, getMostRecentSearchDate } from './searchUtils.ts';
 import { useSavedSearches } from './useSavedSearches.ts';
 
 export const SavedSearchesPage = () => {
-  const [selectedSavedSearch, setSelectedSavedSearch] = useState<SavedSearchesFieldsFragment | null>(null);
-  const [selectedDeleteItem, setSelectedDeleteItem] = useState<number | undefined>(undefined);
   const { t } = useTranslation();
   const { selectedPartyIds } = useParties();
   const { currentPartySavedSearches: savedSearches, isLoading: isLoadingSavedSearches } =
     useSavedSearches(selectedPartyIds);
-  const deleteDialogRef = useRef<DeleteSearchDialogRef>(null);
-  const editSavedSearchDialogRef = useRef<EditSavedSearchDialogRef>(null);
   const formatDistance = useFormatDistance();
   const lastUpdated = getMostRecentSearchDate(savedSearches ?? []) as Date;
+  const queryClient = useQueryClient();
+  const [expandedId, setExpandedId] = useState<string>('');
+
+  const [savedSearchInputValue, setSavedSearchInputValue] = useState<string>('');
 
   if (isLoadingSavedSearches) {
```

Comment:
The SavedSearchesSkeleton should die, right? Its loading state does not align with new design.
 There is a story for `loading` feature on the bookmark section: https://altinn.github.io/altinn-components/?path=/story/bookmarks-bookmarkssection--loading-state